config/routes.rb

```diff
@@ -7,7 +7,7 @@
   end
   resources :donors, only: [:index, :show, :update, :edit]
   resources :events, only: [:create, :index, :show, :update, :edit]
-  resources :users, only: [:create, :show]
+  resources :users, only: [:create, :show, :index, :destroy]
```

Comment:
If possible, let's try to keep the actions in the same order for the different actions. 😀